packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -112,7 +112,14 @@ const commonScreens = (Navigator: typeof Stack) => {
       <Navigator.Screen
         name={Screens.PincodeEnter}
         component={PincodeEnter}
-        options={headerWithBackButton}
+        options={{
+          ...noHeader,
```

Comment:
The fade animation was not working with the header. There is a `headerStyleInterpolator` property but it never faded no matter what I did with it (it just horizontally or vertically swiped). The header animations behave different on all transitions but on this one it was very noticeable and didn't look good.

What I did was remove the back button from the header and add it to the screen itself. This way it has the same fade in/out animation than the rest of the screen.